src/rpc/core_rpc_server.cpp

```diff
@@ -161,6 +162,24 @@ namespace cryptonote
     }
     return true;
   }
+  //------------------------------------------------------------------------------------------------------------------------------
+  bool core_rpc_server::add_host_fail(const connection_context *ctx)
+  {
+    if(!ctx->m_remote_address.is_blockable())
```

Comment:
ctx can be NULL when this is called from inside the daemon (ie, type "status", etc). Unlikely to fail, but still.

Reply:
OK, will check for that